src/pyxplor/pyxplor.py

```diff
@@ -0,0 +1,52 @@
+def time_series_explorer(input_df: pd.DataFrame, 
+                         date_column: str, 
+                         value_column: str, 
+                         freq: str = 'D', 
+                         plot_type: str = 'line', 
+                         figsize: tuple = (10, 6),

```

Comment:
Consistency check: 

**Note that @phchen5 's function has `fig_width` and `fig_height` params**

To adjust one function or the other based on decision

Reply:
Another consistency check:

**Note that @phchen5 's function includes**
```
 label_y_offset: int,
 label_fontsize: int,
```
Q: Do we all want to include this?

Reply:
I agree